y_lb.ws_style_option.yml

```diff
@@ -1,135 +1,135 @@
 y_lb.ws_style_option:
-  - label: Blue Scheme
+  - label: YUSA Blue / Purple
```

Comment:
I guess this is not good to use company name in styles. If we need to mention this is YMCA related, then at least YMCA should be here or I prefer no company at all. Always a problem in the future then

Reply:
@podarok I think I see your concern, but when we're building a distribution that's specifically target to YMCA of the USA but also is frequently used by YMCA of Canada and they have different color options then it might make sense to differentiate them somehow.